src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/BoxSelection.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class BoxSelection : AbstractInteractiveWindowTest
```

Comment:
If this is all interactive window testing, call it InteractiveWindowBoxSelection?